### ISSUE TITLE
Feature-request : Notifications

### ISSUE THREAD

**Issue by CyberB1aze** [SuperioOne/nut_webgui#4]
Not sure if this would be in the scope of this project, but a basic MTX service addon where you can input basic smtp details and have custom notifications send on different states.

**Comment by SuperioOne** [comment]
Yes, it's in the project scope (I actually want something similar for myself).

I'm planning to embed a scripting language like Lua or Rune to provide a generalized plugin system for smtp/webhook notifications and time series exporters, but I can't give a timeline for now.

**Comment by wookiesh** [comment]
Isn't it rather something for nut itself?
